pom.xml

```diff
@@ -79,16 +79,15 @@
         <artifactId>hamcrest</artifactId>
         <version>${hamcrest.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.hamcrest</groupId>
-        <artifactId>hamcrest-library</artifactId>
-        <version>${hamcrest.version}</version>
-      </dependency>
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>
         <exclusions>
+          <exclusion>
```

Comment:
Hamcrest 2,x is contained entirely within the `hamcrest` artifact and both `hamcrest-library` and `hamcrest-core` can be excluded from the transitive dependencies of JUnit 4.x.